.travis.yml

```diff
@@ -15,6 +15,10 @@ sudo: false
 services:
   - docker
 
+env:
+  global:
+    - CC_TEST_REPORTER_ID=3f0e12c83401974ee46f5d1b37beeb818f5acadcd5f7f647d217cd9473c15371
```

Comment:
I googled to see if this ID should be kept secret. The answer is "no". Noting for completeness, no change required. https://docs.codeclimate.com/docs/finding-your-test-coverage-token#should-i-keep-my-test-reporter-id-secret